etc/scripts/orphancompiler.py

```
@@ -0,0 +1,96 @@
+                for elem_id in ids:
+                    if elem_id.startswith('&'):
+                        listed_groups.add(elem_id[1:])
+                    elif '@' not in elem_id:
+                        listed_compilers.add(elem_id)
+            match_aliases = ALIAS_LIST_RE.match(line)
+            if match_aliases:
+                seen_compilers.update(match_aliases.group(1).split(':'))
+                continue
+            match_group = GROUP_NAME_RE.match(line)
+            if match_group:
+                seen_groups.add(match_group.group(1))
+                continue
+            match_compiler = COMPILER_EXE_RE.match(line)
+            if match_compiler:
+                seen_compilers.add(match_compiler.group(1))
+                continue
+            match_disabled = DISABLED_RE.match(line)
+            if match_disabled:
+                disabled.update(match_disabled.group(1).split(' '))
+    bad_compilers = listed_compilers.symmetric_difference(seen_compilers)
+    bad_groups = listed_groups.symmetric_difference(seen_groups)
+    return file, bad_compilers - disabled, bad_groups - disabled
+
+
+def process_folder(folder: str):
+    return [process_file(join(folder, f))
+            for f in listdir(folder)
+            if isfile(join(folder, f))
+            and not (f.endswith('.defaults.properties') or f.endswith('.local.properties'))
+            and f.endswith('.properties')]
+
+
+def find_orphans(folder: str):
+    result = sorted([r for r in process_folder(folder) if len(r[1]) > 0 or len(r[2]) > 0], key=lambda x: x[0])
+    sep = "\n\t"
+    for r in result:
+        print(f'{r[0]}\nCOMPILERS:\n\t{sep.join(sorted(r[1]))}\nGROUPS:\n\t{sep.join(sorted(r[2]))}\n')
+    return result
+
+
+if __name__ == '__main__':
+    if find_orphans('./etc/config/'):
+        print("To suppress this warning on IDs that are temporally disabled, add one or more comments to each listed "
+              "file:")
+        print("# Disabled: id1 id2 ...")
```

So that we can quickly silence the error if needed